lib/Utils.js

```diff
@@ -49,14 +49,15 @@ function unserializeUids(stringdata) {
 */
 function objectCopy(target) {
     const result = target;
-    let source;
-    function copy(f) {
-        result[f] = source[f];
-    }
     /* eslint-disable prefer-rest-params */
-    for (let i = 1; i < arguments.length; i++) {
-        source = arguments[i];
-        Object.keys(source).forEach(copy);
+    const nb = arguments.length;
+    for (let i = 1; i < nb; i++) {
+        const source = arguments[i];
+        const keys = Object.keys(source);
+        const keysNb = keys.length;
+        for (let j = 0; j < keysNb; j++) {
```

Comment:
I wrote a small tests to compare them. In my computer, the new copy function runs faster about 1.5x than the current one.

```
objectCopy 0.00053454461 ms
objectCopy_new 0.00035808709 ms
enhance x 1.4927782233087488 time
```

For comparing for vs. forEach, there are many discussions, here is an example: http://stackoverflow.com/questions/43031988/for-vs-foreach-javascript-efficiency

Source code for testing:
```javascript
const rand = require('crypto').randomBytes;

function objectCopy(target) {
    const result = target;
    let source;
    function copy(f) {
        result[f] = source[f];
    }
    /* eslint-disable prefer-rest-params */
    for (let i = 1; i < arguments.length; i++) {
        source = arguments[i];
        Object.keys(source).forEach(copy);
    }
    /* eslint-enable prefer-rest-params */
    return result;
}

function objectCopy_new(target) {
    const result = target;
    /* eslint-disable prefer-rest-params */
    const nb = arguments.length;
    for (let i = 1; i < nb; i++) {
        const source = arguments[i];
        const keys = Object.keys(source);
        const keysNb = keys.length;
        for (let j = 0; j < keysNb; j++) {
            const key = keys[j];
            result[key] = source[key];
        }
    }
    /* eslint-enable prefer-rest-params */
    return result;
}

function run(fn, loopsNb, target, source1, source2) {
    for (let idx = 0; idx < loopsNb; idx++) {
        fn(target, source1, source2);
        target = {};
    }
}

const data = rand(1024).toString('hex');
const src1 = { name: 'BucketClient' };
const src2 = { bucketName: 'foo0',
  objName: 'keyH5awt100',
  val: `{"key":"keyH5awt100","data":${data}}`,
  path: '/path/to/keyH5awt100' };

function getHrtime(start) {
    const end = process.hrtime(start);
    return (end[0] * 1e3 + end[1] / 1e6);
}

const nb = 1e6;
const start = process.hrtime();
run(objectCopy, nb, {}, src1, src2);
const end = getHrtime(start);
console.log('objectCopy', end / nb, 'ms');

const start2 = process.hrtime();
run(objectCopy_new, nb, {}, src1, src2);
const end2 = getHrtime(start2);
console.log('objectCopy_new', end2 / nb, 'ms');
console.log('enhance x', end / end2, 'time');
```